package.json

```diff
@@ -32,8 +32,10 @@
     "dom-to-image": "^2.6.0",
     "express": "^4.17.1",
     "framer-motion": "^2.6.5",
+    "geoip-lite": "^1.4.2",
```

Comment:
From the viewpoint software architecture adding new npm modules to a project should not be done without intensive questioning of the necessity of this action. 
With every module the complexity of the projects is increased. We have to be sure the used modules are thoroughly maintained, do not add too much weight to our JavaScript bundles and how they might affect the performance of the client code.

Regarding this case: Have we thought about comparing this solution with other solutions?
https://www.npmjs.com/package/geoip-lite, e.g. "NOTE that this requires a lot of RAM. It is known to fail on on a Digital Ocean or AWS micro instance. There are no plans to change this. geoip-lite stores all data in RAM in order to be fast."

Reply:
Do we need a maxmind account and license for this: https://www.npmjs.com/package/geoip-lite#2-update-the-datafiles-optional - and also need to update the data regularly?

Reply:
What does it mean that this part is not static anymore - can it be run on Heroku? 

https://vercel.com/planetapp/planet-webapp/c4pfd418c
```
15:53:44.512 | ┌ ○ /                                                          101 kB          370 kB
15:53:44.512 | ├   └ css/f282e038f85e7897c80f.css                             6.26 kB
15:53:44.512 | ├   /_app                                                      0 B             164 kB
15:53:44.513 | ├ ○ /404                                                       2.86 kB         177 kB
15:53:44.513 | ├ λ /api/geo                                                   0 B             164 kB
15:53:44.513 | ├ ○ /home                                                      2.7 kB          258 kB
15:53:44.513 | ├   └ css/38f0efb0282db38f29c0.css                             5.15 kB
15:53:44.513 | ├ ○ /leaderboard                                               1.54 kB         180 kB
15:53:44.513 | ├ ○ /me                                                        238 kB          507 kB
15:53:44.513 | ├   └ css/973b32407259b4ecdecb.css                             3.84 kB
15:53:44.513 | └ ○ /t/[id]                                                    10.6 kB         267 kB
15:53:44.513 | └ css/89e76bf572df239397c7.css                             2.74 kB
15:53:44.513 | + First Load JS shared by all                                  164 kB
15:53:44.513 | ├ chunks/339dbea8bae5127aa6ae6628fb293040c80c74ae.d929c9.js  58.9 kB
15:53:44.513 | ├ chunks/b1542cc239bf56c8a14913c0a930efe682b5cf86.5f45a7.js  19.3 kB
15:53:44.513 | ├ chunks/b444a69caca13d7f3e0b32cd1e5ad4b3578d4545.00573c.js  17.8 kB
15:53:44.513 | ├ chunks/commons.90c0bf.js                                   11 kB
15:53:44.513 | ├ chunks/framework.f598e0.js                                 40.5 kB
15:53:44.514 | ├ chunks/main.cc2608.js                                      6.46 kB
15:53:44.514 | ├ chunks/pages/_app.c148e9.js                                8.39 kB
15:53:44.514 | ├ chunks/webpack.8238a3.js                                   2.2 kB
15:53:44.514 | └ css/db4c5ce5250358960299.css                               4.86 kB
15:53:44.514 | λ  (Lambda)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
15:53:44.514 | ○  (Static)  automatically rendered as static HTML (uses no initial props)


```

Reply:
I just tested it with https://planet-webapp-git-feature-configapiinnext.planetapp.vercel.app/ on Vercel:

<img width="807" alt="Bildschirmfoto 2020-10-05 um 16 28 37" src="https://user-images.githubusercontent.com/1532418/95092425-ce4f9700-0751-11eb-88f9-a9776af14b69.png">

I am located in Hamburg, Germany - this module has calculated a location for me 400km away ;-)

Reply:
@norbertschuler this is a test project; and we might not use it at all.

Also, This is fallback case; when CF IP Country is not available; eg: tenant does not use Cloudflare.

Reply:
@harshvitra ok, so I tested it :-)